README.md

```diff
@@ -46,39 +29,40 @@ The following dependencies are required and must be installed separately.
 
 ### Install via `go get`
 
-To install Annie, use `go get`, or download the binary file in the [Releases](https://github.com/iawia002/annie/releases) page.
+To install Annie, use `go get`, or download the binary file from [Releases](https://github.com/iawia002/annie/releases) page.
 
 ```bash
 $ go get github.com/iawia002/annie
-...
-$ annie [args] URL
 ```
 
 
-## Get Started
+## Getting Started
 
 ### Download a video
 
 ```console
-$ annie -s 127.0.0.1:1080 https://youtu.be/Gnbch2osEeo
+$ annie 'https://www.youtube.com/watch?v=v-Dur3uXXCQ'
 
- Site:      YouTube youtube.com
- Title:     Multifandom Mashup 2017
+Site:      YouTube youtube.com
+ Title:     Kesha - Praying (Official Video)
  Type:      video
- Stream:
+ Stream:   
      [default]  -------------------
      Quality:         hd720
-     Size:            57.97 MiB (60785404 Bytes)
+     Size:            50.62 MiB (53080675 Bytes)
      # download with: annie -f default "URL"
 
- 11.93 MiB / 57.97 MiB [======>-------------------------]  20.57% 19.03 MiB/s 2s
+
+ 11.93 MiB / 50.627 MiB [=======>------------------------]  23.6% 18.53 MiB/s 2s
 ```
 
 > Note: wrap the URL in quotation marks if it contains special characters. (thanks @tonyxyl for pointing this out)
 >
 > `$ annie 'https://...'`
 
-The `-i` option displays all available formats information without downloading.
+## Usage
```

Comment:
no `Usage` here, because the whole structure is like this:
```
Getting Started
  * Download a video
  * Download anything else
  * Download playlist
...
```

this `Usage` will break this structure.